### ISSUE TITLE
More Flexibility with Specialized Commands

### ISSUE THREAD

**Issue by jiajunmao** [Zrips/CMI#5338]
Hey, so I think for now we are able to use perm:[permNode] to check whether a player has a perm node in specialized commands. Is it possible to combine that with the allPlayers! directive such that for all the players, the ones with or without certain perms would execute certain commands.
The same combination can go together with other constraint checking such as hasMoney.

**Comment by CloudeLecaw** [comment]
I agree, checks like this would be usefull some time.

**Comment by jiajunmao** [comment]
Yeah, for now we can kinda hack it through creating like a custom alias and do the perm check there. So like having allPlayer wrapper calling the aliased command. But being able to have them together in one place is kinda nice.
